connector/doc/guides/jobrunner.rst

```diff
@@ -19,13 +19,20 @@ The ``ODOO_CONNECTOR_CHANNELS`` environment variable must be
 set before starting Odoo in order to enable the job runner
 and configure the capacity of the channels.
 
+Alternatively, set the channel configuration in the Odoo configuration file:
+
+.. code-block:: cfg
+
+    [options-connector]
+    channels = root:4
```

Comment:
It would be easier if we can define channels this way:
```
[options-connector]
channels: 
  root: 4,
  root.connector-magento: 2
...
```
(I think they are standard configuration files options)

Is that possible?